ui/plugins/pluginpanel.html

```diff
@@ -116,6 +107,24 @@
             </a>
           </li>
         </ul>
+        <ul class="nav navbar-nav navbar-right">
+          <li>
+            <button
+               class="dropdown-toggle icon-blue"
```

Comment:
As mentioned in my general PR testing comments, if we make this consistent with our other action bar buttons (`fa-angle-down`, extra dark background, extra light grey, on hover), we'd remove this `icon-blue` here. I think we could also purge `icon-cyan` from treeherder-global.css, since the Backfill button was the only UI that used it so far. https://github.com/mozilla/treeherder/blob/676e1f4738dd2ddc8fcf6aeb525d7fb13409ae73/ui/css/treeherder-global.css#L112-L116